apps/framework-docs/src/pages/on-prem/configuring-moose-for-cloud.mdx

```diff
@@ -1,14 +1,14 @@
 # Configuring Moose for cloud environments
 
-In the [Packaging Moose for deployment](packaging-moose-for-deployment.mdx) page we looked at how to package your moose application into docker containers (using the `moose build --docker` command) and you've pushed them to your container repository.
+In the [Packaging Moose for deployment](packaging-moose-for-deployment.mdx) page, we looked at how to package your moose application into Docker containers (using the `moose build—-docker` command), and you've pushed them to your container repository.
```

Comment:
maybe keep the space between `moose build --docker`

Reply:
Nice catch!